src/Kinepolis/ParsedMovie.php

```diff
@@ -27,14 +27,17 @@ final class ParsedMovie
 
     private PriceInfo $priceInfo;
 
+    private string $poster;
+
     public function __construct(
         string $externalId,
         Title $title,
         LocationId $locationId,
         Description $description,
         Theme $theme,
         Calendar $calendar,
-        PriceInfo $priceInfo
+        PriceInfo $priceInfo,
+        string $poster
```

Comment:
What is poster? Is that the URL of the image?

Reply:
Indeed,  I kept the naming consistent with the naming in the JSON API from Kinepolis.

Reply:
I have no strong opinion about it, but is image more clear in the UDB3 context?